src/components/projectSelect/index.tsx

```diff
@@ -83,3 +119,31 @@ export const ProjectSelect: React.FC<ProjectSelectProps> = ({
     />
   );
 };
+
+const { reportError } = errorReporting;
+
+export const getRepoId = (projects: Project[]) => {
+  if (!projects || projects.length === 0) {
+    reportError(
+      new Error("No projects provided when fetching repo id")
```

Comment:
@khelif96 this error shows up in the test. I don't think we should error here because there is probably just a point where it's undefined and it'll probably keep logging.

Reply:
If thats the case then it would also show up in production. So maybe we need to guard against this case better?

Reply:
I don't think it'll ever be a problem, it'll be there by the time the page is loaded.